wiki/Medals/Unlock_requirements/Seasonal_spotlights/de.md

```diff
@@ -0,0 +1,11 @@
+# Freischaltungsanforderungen für Seasonal Spotlights Medaillen
+
+| Medaillenname | Anforderung |
+| :-- | :-- |
+| Summer 2018 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einer der Beatmap-Spotlights enthalten ist: Beatmap-Pakete von Sommer 2018 ([osu!](https://osu.ppy.sh/beatmaps/packs/1508)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1509)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1510)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1511)). |
+| Fall 2018 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einer der saisonalen Spotlights enthalten ist: Beatmap-Pakete von Herbst 2018 ([osu!](https://osu.ppy.sh/beatmaps/packs/1548)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1549)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1550)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1551)). |
+| Winter 2019 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einer der saisonalen Spotlights enthalten ist: Beatmap-Pakete von Winter 2019 ([osu!](https://osu.ppy.sh/beatmaps/packs/1623)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1624)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1625)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1626)). |
+| Spring 2019 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einer der saisonalen Spotlights enthalten ist: Beatmap-Pakete von Frühling 2019 ([osu!](https://osu.ppy.sh/beatmaps/packs/1670)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1671)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1672)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1673)). |
+| Summer 2019 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einer der saisonalen Spotlights enthalten ist: Beatmap-Pakete von Sommer 2019 ([osu!](https://osu.ppy.sh/beatmaps/packs/1722)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1723)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1724)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1725)). |
+| Autumn 2019 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einer der saisonalen Spotlights enthalten ist: Beatmap-Pakete von Herbst 2019 ([osu!](https://osu.ppy.sh/beatmaps/packs/1798)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1799)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1800)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1801)). |
+| Winter 2020 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einer der saisonalen Spotlights enthalten ist: Beatmap-Pakete von Winter 2020 ([osu!](https://osu.ppy.sh/beatmaps/packs/1896)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1897)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1898)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1899)). |
```

Comment:
```suggestion
| Summer 2018 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einem der Beatmap-Spotlights-Pakete von Sommer 2018 enthalten sind ([osu!](https://osu.ppy.sh/beatmaps/packs/1508)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1509)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1510)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1511)). |
| Fall 2018 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einem der saisonalen Spotlights-Pakete von Herbst 2018 enthalten sind ([osu!](https://osu.ppy.sh/beatmaps/packs/1548)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1549)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1550)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1551)). |
| Winter 2019 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einem der saisonalen Spotlights-Pakete von Winter 2019 enthalten sind ([osu!](https://osu.ppy.sh/beatmaps/packs/1623)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1624)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1625)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1626)). |
| Spring 2019 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einem der saisonalen Spotlights-Pakete von Frühling 2019 enthalten sind ([osu!](https://osu.ppy.sh/beatmaps/packs/1670)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1671)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1672)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1673)). |
| Summer 2019 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einem der saisonalen Spotlights-Pakete von Sommer 2019 enthalten sind ([osu!](https://osu.ppy.sh/beatmaps/packs/1722)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1723)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1724)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1725)). |
| Autumn 2019 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einem der saisonalen Spotlights-Pakete von Herbst 2019 enthalten sind ([osu!](https://osu.ppy.sh/beatmaps/packs/1798)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1799)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1800)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1801)). |
| Winter 2020 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einem der saisonalen Spotlights-Pakete von Winter 2020 enthalten sind ([osu!](https://osu.ppy.sh/beatmaps/packs/1896)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1897)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1898)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1899)). |
```
I think this is more correct.

Reply:
I agree with you on `enthalten sind`, due to the plural of beatmaps. However, I think my translation is closer to the original: `Clear all beatmaps bundled in one of the Beatmap Spotlights: Summer 2018 beatmap packs`. What do you think about this?

Reply:
Hmm yeah, the sentence sounds a bit weirdly worded to me in both languages, but i guess it's fine. updated the suggestion.